src/common/policy.c

```diff
@@ -0,0 +1,130 @@
+/**
+ * (C) Copyright 2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+/**
+ * This file is part of daos. It implements some miscellaneous policy functions
+ */
+
+#include "policy.h"
+#include <string.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+#define POLICY_BAD_PARAM	(-1)
+
+static const char *POLICY_NAMES[DAOS_MEDIA_POLICY_MAX] = {
+	"io_size",
+	"write_intensivity" };
+
+static const char *POLICY_PARAMS[DAOS_MEDIA_POLICY_MAX]
+				[DAOS_MEDIA_POLICY_PARAMS_MAX] =
+{
+{"th1", "th2", "", ""},				/* io_size */
+{"wr_size", "hot1", "hot2", ""}			/* write_intensivity */
+};
+
+static const char *PARAM_DELIM = ",";
+static const char *VALUE_DELIM = "=";
+
+bool
+daos_policy_try_parse(const char *policy_str, struct policy_desc_t *out_pd)
+{
+	char *tok		= NULL;
+	const char *delim	= PARAM_DELIM;
+	const char *delim2	= VALUE_DELIM;
+	char *saveptr		= NULL;
+	char *saveptr2		= NULL;
+	char *str2		= NULL;
+	bool type_found		= false;
+	int param_idx		= POLICY_BAD_PARAM;
+	int tok_len		= 0;
+	bool ret_val		= false;
+
+	int i,j,k;
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)